sections/main-product.liquid

```diff
@@ -359,6 +359,15 @@
                   {{ product.description }}
                 </div>
               {%- endif -%}
+            {%- when 'sku' -%}
```

Comment:
Since we're surfacing new information when a variant is selected, we should probably announce that to screen readers too. We do this with the product's price, which has a `role="status"`. We can use a similar approach here.